docs/mobile-apps/app-storage.md

```diff
@@ -227,6 +227,43 @@ caps.SetCapability("app","storage:c8511dd6-38ec-4f58-b8b9-4ec8c23ad882");
 </TabItem>
 </Tabs>
 
+## Using Dependent Apps for a Test
+
+<small><span class="sauceDBlue">Real Devices Only</span></small>
+
+If your real device testing requires your app under test to have access to other apps and you, therefore, need to install those dependent apps and reference them in your tests, you can do so using the `otherApps` capability.
+
+:::note
+Dependent apps inherit the configuration of the main app under test for settings such as `proxy`, `locale`, and `device orientation`, regardless of what settings may have been applied to the app at the time of upload, because the settings are specific to the device under test.
+:::
+
+### Appium Capability
+
+For [Appium](/dev/test-configuration-options#mobile-app-capabilities-sauce-specific--optional) tests, you can specify up to seven apps that have already been uploaded to App Storage using one of the previously described methods by setting the `otherApps` desired capability and referencing the app's storage ID or filename.
+
+```
+caps.setCapability("otherApps", "storage:filename=<file-name>")
+caps.setCapability("otherApps", "storage:fileid=<file-id>")
+```
+
+:::note
```

Comment:
@JudgeSi @sweeneyskirt-sl  it probably is important to add here the app settings and device settings that the dependant apps inherit? Such as the locale, orientation, etc.?

Reply:
Yes, sounds like a good idea, maybe we can link to a section which explains these device settings, does this section exist yet?

Reply:
I am of the opinion that the note in the preceding section regarding that inheritance is sufficient. I put in the "parent" section because it applies to both the Appium and the Espresso/XCUITest sections. Are you thinking that info needs to be repeated here (and in the Espresso/XCUITest section), or should it be moved, or am I missing the point and you are looking for something else to be more explicit? Just want to make sure I understand the ask. :)